### PR TITLE
Add additional heatmap schemas

### PR DESCRIPTION
**Pull request by jerowe**
References: https://github.com/apache/incubator-superset/issues/4574
ping @mistercrunch

### CODE REVIEW

**Comment by jerowe** [comment]
I'm not done, do not merge!

**Comment by mistercrunch** [comment]
Mind sharing screenshots of what the new options look like?

**Comment by jerowe** [comment]
In theory I could, but somehow I broke the npm build and I'm trying to get it working again. Once I get it working I am happy to post screenshots.

**Comment by jerowe** [comment]
Pinging @mistercrunch 

screenshot 2018-08-06 12 16 31
screenshot 2018-08-06 12 15 44
screenshot 2018-08-06 12 15 26
screenshot 2018-08-06 12 11 19

**Comment by jerowe** [comment]
@mistercrunch  - I was thinking about changing some of the d3 colorschemes so that the first color (which is close to white) is dropped, but it seems that now missing blocks are gray. So maybe its not such a big deal?

**Comment by jerowe** [comment]
Hmmm I'm getting an error when I navigate to a dashboard, but I don't know why.

```
Uncaught TypeError: Cannot read property 'children' of undefined
    at t.default (dashboard.b259de2c9caaee2d7265.entry.js:111)
    at t.default (dashboard.b259de2c9caaee2d7265.entry.js:111)
    at Object.<anonymous> (dashboard.b259de2c9caaee2d7265.entry.js:102)
    at s (dashboard.b259de2c9caaee2d7265.entry.js:1)
    at Object.<anonymous> (dashboard.b259de2c9caaee2d7265.entry.js:102)
    at s (dashboard.b259de2c9caaee2d7265.entry.js:1)
    at n (dashboard.b259de2c9caaee2d7265.entry.js:1)
    at dashboard.b259de2c9caaee2d7265.entry.js:1
    at dashboard.b259de2c9caaee2d7265.entry.js:1
```

**Comment by mistercrunch** [comment]
@jerowe there was a bug on `master`, rebase your branch should fix that, also please remove the `[WiP]` from the PR title if ready for review

**Comment by jerowe** [comment]
I will wait for the tests to complete and then remove the WIP.

**Comment by jerowe** [comment]
ping @mistercrunch (sorry to be a pain but I really want this.)

**Comment by jerowe** [comment]
Ok and as long as  we are talking about stuff I want:

Color select for missing values
Color select for values that don't make some cutoff (I think this will be harder since I always do the cutoff through sql )

**Comment by mistercrunch** [comment]
Indents seem off, should be 2 not 4. I don't understand why the linter doesn't fire on that.

**Comment by jerowe** [comment]
I tried to get my editor to go along with the project configs, but didn't quite get it perfect. Any tips? I use webstorm

**Comment by mistercrunch** [comment]
Use vim :)

**Comment by jerowe** [comment]
@mistercrunch , I fixed the indentation issues.

**Comment by mistercrunch** [comment]
@elibrumbaugh @williaster what do you think of adding these color schemes?

**Comment by jerowe** [comment]
ok now it is all ready

File: superset/assets/src/explore/controls.jsx [hunk 1]
```diff
@@ -324,6 +324,40 @@ export const controls = {
       ['red_yellow_blue', 'red/yellowish/blue'],
       ['brown_white_green', 'brown/white/green'],
       ['purple_white_green', 'purple/white/green'],
+      ['schemeBrBG', 'd3BrBG'],
```
Comment (williaster) [inline superset/assets/src/explore/controls.jsx]: can you please make these more human readable? non-power users are not going to have any idea what `BrBG` means

**Comment by williaster** [comment]
I'm fine with it, this isn't the long term solution we want but fine for now 🤷‍♀️

**Comment by kristw** [comment]
LGTM. I agree with @williaster 
@jerowe I would recommend squashing your git commits and clean the commit messages a little bit as these messages will forever be stored in the main repo git log. 
https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

File: superset/assets/src/explore/controls.jsx (hunk 1 above)
Comment (jerowe) [reply superset/assets/src/explore/controls.jsx]: sure. updated.

**Comment by jerowe** [comment]
@kristw , I just tried to squash my commits, but I'm not sure I was successful. Can they get squashed on merge?

**Pull request merged**